builtins/amp-video.md

```diff
@@ -82,9 +83,10 @@ Similar to the `video` tag `controls` attribute - if present, the browser offers
 
 If present, will automatically loop the video back to the start upon reaching the end.
 
-**muted**
+**muted (deprecated)**
```

Comment:
OK, I don't completely follow the logic, since having a standard attribute
just do what it does seems easier to document than anything else, but don't
want to bikeshed it. My recommendation would be to say: muted, OK, this
video will not have sound.

Let me know when the rest the other change are made.

On Mon, Sep 12, 2016 at 4:37 PM, Ali Ghassemi <notifications@github.com>
wrote:

> In builtins/amp-video.md
> <https://github.com/ampproject/amphtml/pull/4786#discussion_r78474467>:
>
> > @@ -82,9 +83,10 @@ Similar to the `video` tag `controls` attribute - if present, the browser offers
> >
> >  If present, will automatically loop the video back to the start upon reaching the end.
> >
> > -**muted**
> > +**muted (deprecated)**
>
> yeah and that case is covered by autoplay and lack of controls. <amp-video
> autoplay loop> would exactly do what you are describing.
>
> What we are missing now is ability to autoplay with sound which would only
> work in desktop. I suggest not worrying about that case yet, if we need to
> have it, we can add support either using autoplay=withsound or
> autoplay-with-sound. More interesting (and more challenging) would be
> trying to carry user's intention of watching a video all the way from the
> platform in a way that browser would allow us to call play() on unmuted
> videos to get unmuted autoplay in mobile.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ampproject/amphtml/pull/4786/files/c2a77b4669e07783c5adc2dc14b5835b7bb91884#r78474467>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFeT1Qvi4M4WDwV0Hppe5eWCwCcQIkTks5qpeIjgaJpZM4JxI8M>
> .
>